src/routes/blog/post/deploy-nextjs-app-to-appwrite-sites/+page.markdoc

```diff
@@ -0,0 +1,72 @@
+
+To build your Next.js app, run the following command in your terminal:
+
+```bash
+npm run build
+```
+
+If your build succeeds, you can deploy your Next.js application to the internet. If your build fails, you must fix all the errors in your application. When deploying a Next.js app, the deployment provider builds your app on their servers, and cannot deploy if the build fails.
+
+# Configuring your Site on Appwrite
+
+This article assumes you have your Next.js app code stored in a GitHub repository. If not, please do so, as it is needed for deployment. Deploying through code stored on GitHub also introduces advantages such as automatic deployments, preview deployments, etc.
+
+Now, let’s link your GitHub repository to Appwrite Sites.
+
+- Go to [Appwrite Console](https://cloud.appwrite.io) and sign in using your preferred authentication method.
+- Create a new project, or use an existing one.
+- Click on the **Sites** option on the sidebar.
+- Click on **Create Site**.
+- Click on **Connect a GitHub repository**.
+
+Now, you should be able to see the most recent GitHub repositories based on activity.
+
+![Connecting your GitHub repository to Appwrite Sites](/images/blog/deploy-nextjs-app-to-appwrite-sites/connect-repo.png)
+
+Click **Connect** against the desired GitHub repository. The Next.js logo should be the icon of any valid Next.js app.
+
+Once you connect your GitHub repository, you will be presented with options for deployment.
+
+![Configuring your Site on Appwrite](/images/blog/deploy-nextjs-app-to-appwrite-sites/site-config.png)
+
+Fill in the details as desired. Although there are some things you must know before moving forward.
+
+- If you’re using a different branch for your intended production website, choose the specific branch.
+- If you’re using a monorepo, Sites might not automatically detect your application. In such cases, manually provide the root directory for your Next.js app that contains the Next.js configuration file. For example, if you’re using Turborepo, your Next.js app might be located inside one of the folders in the `apps` folder, so your root directory should be something like `./apps/web`.
+- Most users don’t need to change the **Build settings**, although if you want to run different commands for installation and builds, you can configure those in this section. If you don’t know anything about this, it’s worth skipping this section altogether.
+- If your app uses environment variables, add them in the **Environment variables** section. If your app uses sensitive API keys, you are likely using environment variables.
+    - In Next.js, public environment variables start with `NEXT_PUBLIC_`. These environment variables are accessible in the browser environment, so don’t store your sensitive API keys in a variable that starts with that prefix.
```

Comment:
Instead of a subpoint, you can also use an info note (check the CONTENT.md file)